todo_app/templates/index.html

```diff
@@ -89,13 +89,13 @@ <h5>Update Item</h5>
       <label for="title">Updated status: </label>
         <div class="btn-group btn-group-toggle" data-toggle="buttons">
           <label class="btn btn-secondary active">
-            <input type="radio" name="status" id="option1" autocomplete="off" value="64ae6f476f946f8c451a5cb9" checked> Todo
+            <input type="radio" name="status" id="option1" autocomplete="off" value="todo" checked> Todo
```

Comment:
GitHub won't allow me to easily comment on unchanged code, but if we did want to use IDs rather than names, we can update the code on lines 56/61/66 to something like:
```html
<option value="{{item.id}}">{{item.name}}</option>
```
So that the user is still choosing between the human-readable names, but the selected data sent to the backend is the id